compysition/actor.py

```diff
@@ -206,9 +205,12 @@ def _loop_send(self, event, queues, check_output=True):
         if check_output and not isinstance(event, self.output):
             raise InvalidActorOutput("Event was of type '{_type}', expected '{output}'".format(_type=type(event), output=self.output))
 
-        if len(queues) > 0:
-            self._send(queues[0], deepcopy(event))
-            map(lambda _queue: self._send(_queue, deepcopy(event)), queues[1:])
+        if isinstance(queues, _InternalQueuePool):
```

Comment:
I would look into a try/except block instead of isinstance. Python style is usually forgiveness as opposed to permission.